src/expression/embeddedDocs/function/statistics/cumsum.js

```diff
@@ -0,0 +1,24 @@
+export const cumSumDocs = {
+  name: 'cumsum',
+  category: 'Statistics',
+  syntax: [
+    'cumsum(a, b, c, ...)',
+    'cumsum(A)'
+  ],
+  description: 'Compute the sum of all values.',
+  examples: [
+    'cumsum(2, 3, 4, 1)',
+    'cumsum([2, 3, 4, 1])',
+    'cumsum([2, 5; 4, 3])'
+  ],
+  seealso: [
+    'max',
```

Comment:
Should I add a `'cumsum'` to the `seealso`s in other files here? @josdejong

Reply:
Yes, good idea, would be nice to add a reference for example in `sum`.